problemsets-mat223/modules/module6.tex

```diff
@@ -54,7 +54,49 @@
 	Let $\mathcal V\subseteq \R^2$ be the complete solution to 
 	$x+2y=0$.  Show that $\mathcal V$ is a subspace.
 
-	XXX Finish
+	Let $\vec u = \mat{u_1\\u_2}$, $\vec v = \mat{v_1\\v_2} \in \mathcal V$.
```

Comment:
You shouldn't combine `=` with `\in`. How about, `Let $\vec u = \mat{u_1\\u_2}$ and $\vec v = \mat{v_1\\v_2} be in $\mathcal V$`